internal/appsec/waf/waf.go

```diff
@@ -223,73 +262,81 @@ type Context struct {
 // NewContext a new WAF context and increase the number of references to the WAF
```

Comment:
Sorry for being picky, can we add a "creates" or "initializes", or something similar here?